README.md

```diff
@@ -151,6 +152,17 @@ status, input and both outputs to it:
 Cheetah.run("ls -l", logger: logger)
 ```
 
+### Overwriting env
+
+If the command needs adapted environment variables, use the :env option.
+Passed hash is used to update existing env (for details see ENV.update).
+Nil value mean unset varible. Environment is return back to original state after
```

Comment:
"variable" missing "a"

"Environment is restored to its original state after running the command"